probeinterface/probe.py

```diff
@@ -931,6 +945,34 @@ def _2d_to_3d(data2d, plane):
         raise ValueError('Bad plane')
     return data3d
 
+def data_select_dimensions(data, plane='xy'):
+    """
+    Select dimensions in a 3d or 2d array.
+    
+    Parameters
+    ----------
+    data: np.array
+        shape (n, 2) or (n, 3)
+    plane: str
+        'xy', 'yz' 'xz' or 'xyz'
```

Comment:
this can be any combination of these. We shoudl check only that `len(np.unique(dimensions)) == len(dimensions)`

Reply:
yep.